nameko/containers.py

```diff
@@ -29,7 +29,12 @@
 
 
 def get_service_name(service_cls):
-    return getattr(service_cls, "name", service_cls.__name__.lower())
+    service_name = getattr(service_cls, 'name', None)
+    if service_name is None:
+        raise RuntimeError('Service class must define a `name` attribute')
```

Comment:
do we want some custom `ConfigurationError`? (e.g. to catch and display something nicer than a traceback in `nameko run`) or are we happy with a traceback in this case?